### PR TITLE
Allows users to belong to publishers with a role.

### PR DESCRIPTION
**Pull request by rossjones**
The many_to_many is messy because it's not really complete in Ecto, it's
too complex to get the extra columns (in this case :role) from the join
table so I have hacked around it in the user model.

### CODE REVIEW

File: src/dguweb/web/templates/user/index.html.eex [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+<h1 class="heading-medium">My publishers</h1>
```
Comment (maxf) [inline src/dguweb/web/templates/user/index.html.eex]: Elsewhere I put heading-xlarge. Can you change so they're all the same? I slightly lean towards medium or large

File: src/dguweb/test/controllers/user_controller_test.exs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    conn = post conn, session_path(conn, :login), session: @valid_attrs
+    assert redirected_to(conn) == "/user"
+
+    conn = get conn, user_path(conn, :index)
+    assert html_response(conn, 200) =~ "Cabinet Office" 
+    assert html_response(conn, 200) =~ "admin" 
+  end 
+
+
+end
```
Comment (maxf) [inline src/dguweb/test/controllers/user_controller_test.exs]: convention is line breaks at the end of files.

File: src/dguweb/priv/repo/seeds.exs [hunk 3]
```diff
@@ -96,9 +97,10 @@ R.insert(%T{
 
 # Themes ######################################################################
 
+R.delete_all PU
 R.delete_all P
 
-R.insert(%P{
+cabinet_office = R.insert(%P{
```
Comment (maxf) [inline src/dguweb/priv/repo/seeds.exs:103]: are those vars used anywere?

Comment (rossjones) [inline src/dguweb/priv/repo/seeds.exs:103]: In the next upcoming PR ;)

**Pull request merged**